tests/integration/test_article_upload.py

```diff
@@ -14,6 +14,12 @@
 from tests.responses import get_response_dir, read_hep_schema, read_titles_schema, read_response
 
 
+def test_empty_record():
```

Comment:
I'd be more descriptive :)

Reply:
If you want to really test the `update_record` function, you should
* insert a record in the DB and ES
* create a wf with real data
* invoke `update_record` passing the wf
* check that the record you created previously has the new data

In case you want to test further, you can try to create the workflow with some bad data in order to trigger the `ValidationError` and checking that the workflow is actually halted.